onnxruntime/contrib_ops/rocm/bert/fast_gelu_tunable_op.h

```diff
@@ -34,7 +34,9 @@ struct FastGeluParams : OpParams {
 template <typename T, int ThreadsPerBlock, int VecSize>
 Status FastGeluOp(const FastGeluParams<T>* params) {
   // TODO(anyone): Add tail handling for FastGelu
-  TUNABLE_OP_RETURN_UNSUPPOTED_ARGUMENT_IF(!(params->bias_length > 0 && (params->bias_length % VecSize == 0)));
+  TUNABLE_OP_RETURN_UNSUPPOTED_ARGUMENT_IF(
+      !((params->bias_length > 0 && params->bias_length % VecSize == 0 && params->input_length % VecSize == 0) ||
+        params->input_length % VecSize == 0));
```

Comment:
Sorry, just realized it's not fully correct, should be:

(params->bias_length > 0 && params->bias_length % VecSize == 0 && params->input_length % VecSize == 0) ||
        (params->bias_length == 0 && params->input_length % VecSize == 0))